packages/router/src/utils/createBrowserHistory.ts

```diff
@@ -103,7 +104,7 @@ const createBrowserHistory = (win: Window, props: CreateBrowserHistoryOptions =
     );
   };
 
-  const handlePopState = (event: any) => {
+  const handlePopState = (event: PopStateEvent) => {
```

Comment:
Misc. clean up discovered while reading through code.  Seemed like a specific type would be better.